spec/javascripts/content_links_spec.js

```diff
@@ -1,5 +1,6 @@
 describe("PopupView.generateContentLinks", function () {
-  var PROD_ENV = { protocol: 'https', serviceDomain: 'publishing.service.gov.uk' }
+  var PROD_ENV = { protocol: 'https', serviceDomain: 'publishing.service.gov.uk', origin: 'https://www.gov.uk' }
+  var DRAFT_ENV = { protocol: 'https', serviceDomain: 'integration.publishing.service.gov.uk', origin: 'https://draft-origin.integration.publishing.service.gov.uk' }
```

Comment:
Normally the terms "draft" and "live" are used when referring to draft content, and live content.

It looks here like the environment is Integration, and the origin is the draft one, not the live one.

Reply:
Would you rather I changed it to https://draft-origin.publishing.service.gov.uk?

Reply:
I should have been more specific, I was more referring to `DRAFT_ENV` being the variable name. At least in my mind, `DRAFT` isn't in the same category as `PROD`, so while there's a pattern in the variable names, it's confusing to me.

Reply:
Ah, I see what you mean. Agreed - shall I change the URL as suggested and rename to `DRAFT_PROD_ENV`?

Reply:
Yeah, that sounds sensible :+1: